site/en/docs/devtools/css/color/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Inspect and debug HD and non-HD colors with the Color Picker"
+authors:
+  - jecelynyeen
+  - sofiayem
+date: 2023-03-07
+#updated: YYYY-MM-DD
+description: "Learn how to use the Color Picker in Elements > Styles to inspect and debug HD and non-HD colors."
+tags:
+  - prototype-fixes
+  - css
+---
+
+The **Color Picker** provides a GUI for changing `color` and `*-color` declarations and lets you create, convert, and debug non-HD and [HD colors](/articles/high-definition-css-color-guide/) with a click.
+
+{% YouTube id='TuR27BxCRVk' %}
+
+{% Aside "important" %}
+This video shows Chrome DevTools version earlier than 112. In later versions, the **Styles** pane supports:
+
+- 12 new color spaces and 7 new gamuts from the [CSS Color Level 4](https://www.w3.org/TR/css-color-4/) specification.
+- The [`color-mix()`](/blog/css-color-mix) function from [CSS Color Level 5](https://www.w3.org/TR/css-color-5/#color-mix).
+{% endAside %}
+
+For a deep dive on the new color spaces, see [High Definition CSS Color Guide](/articles/high-definition-css-color-guide/).
+
+## Open the Color Picker and change colors {: #change-colors }
```

Comment:
Somewhere in this docs, mention that user can Shift + Click on the color preview icon to open the conversion dialog

Reply:
I did in two instances.